Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/ResourceClasses/Constants.java

```diff
@@ -0,0 +1,9 @@
+package org.firstinspires.ftc.team6220_PowerPlay.ResourceClasses;
+
+public class Constants {
+    public double WheelDiam = 3.73;
+    public double WheelCirc = 11.71;
```

Comment:
You should derive the wheel circumference and inches per tick from the diameter.  That way, you only need to change the one parameter - diameter if you change to another size wheel.